src/RA_Core.cpp

```diff
@@ -801,7 +801,7 @@ API HMENU CCONV _RA_CreatePopupMenu()
     {
         // TODO: Use the literals once PR 23 is accepted
         AppendMenu(hRA, MF_STRING, IDM_RA_FILES_LOGOUT, TEXT("Log&out"));
-        AppendMenu(hRA, MF_SEPARATOR, UINT_PTR{}, nullptr);
+        AppendMenu(hRA, MF_SEPARATOR, 0U, nullptr);
```

Comment:
The only thing that might matter (unsigned) short. The other stuff shouldn't.
Which one seems more desirable? They basically all resolve the same value.
```
short{};
(short)0;
static_cast<short>(0);
```

Reply:
My old TODO comments is gone.